skops/card/_model_card.py

```diff
@@ -205,20 +206,19 @@ def save(self, path: str | Path) -> None:
 
         card.save(path)
 
-    def _extract_estimator_config(self) -> str:
+    def _write_hyperparameters(self) -> str:
```

Comment:
It doesn't really matter too much, but I'm not 100% convinced about `_write_hyperparameters`. "Write" for me implies that the hyperparameters are written somewhere, e.g. to a file.